### ISSUE TITLE
/ps unstuck, to unstuck players from areas they they do not own.

### ISSUE THREAD

**Issue by Leonardo-Fiori** [espidev/ProtectionStones#60]
Add a /ps unstuck command or just /unstuck.

It would teleport players at the border of the world guard regions in which they are standing.

It would be useful for players when they are stuck inside a region that they do not own, in servers (like mine) where /tp /tpa /warp are not allowed.

A plugin like that already exists, but it does not always works and it overrides world guard's blocked-cmds flag.

It would be a nice feature to have on Protection Stones to make it a 100% great claiming plugin!

**Comment by espidev** [comment]
The algorithm for this to not be buggy might be hard, will have to investigate

**Comment by espidev** [comment]
Have you found a plugin that works for this?

**Comment by Leonardo-Fiori** [comment]
Kinda.. it's called region unstuck. But it's not working all that good. Sometimes it fails, and it ignored wg blocked cmds flag

**Comment by espidev** [comment]
Yeah, I'm not sure if I can do a better job, but I'll have to find time to sit down and tackle this in the future (not the highest priority right now)
